runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -239,4 +239,15 @@ default boolean swapDecant()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		position = 19,
+		keyName = "swapLever",
+		name = "Lever",
+		description = "Swap Ardougne with Edgeville for the wildnerness teleport lever."

```

Comment:
wildnerness -> Wilderness

Reply:
haha just noticed that, fix should be up any second now